spec/controllers/scim_rails/scim_users_controller_spec.rb

```diff
@@ -231,6 +231,8 @@
         http_login(company)
       end
 
+      let(:user) { company.users.first }
```

Comment:
`let`s shouldn't be off on their own like this.
Try to keep them at the top of whatever context/describe block they're in (with subject first) just so there's no "surprises" for people trying to understand the specs